dtcw

```diff
@@ -417,6 +408,11 @@ if [ "${doJavaCheck}" = true ] ; then
     check_java
 fi
 
-# echo "Command to invoke: ${command}"
+# use x86 emulator on Apple M1/M2
+emu=`[ $os = "Darwin" ] && [ $arch = "arm64" ] && echo "arch -x86_64"` 
```

Comment:
can we formulate this a bit more readable?

```
emu=""
if $os = "Darwin" && $arch = "arm64" ; then
   emu="arch -x86_64"
fi
```

(don't know if this is the correct syntax)

this is longer but I guess it helps to maintain the source

Reply:
sure, fine for me - I'll change it a bit to make it more readable

Reply:
... done ;-)